src/app/components/header/header.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="header">
+    <div class="header-content container">
+        <div class="header-logo-label">
+            <div class="header-logo"><img alt="My image name" src="../../../assets/icons/logo.svg"></div>
```

Comment:
It is good to have const variables to keep strings in a separate file and reuse them where needed.

Reply:
I have used const variables through service in the resolve commit apu.